src/platform/forms/save-in-progress/api.js

```diff
@@ -1,41 +1,22 @@
 import * as Sentry from '@sentry/browser';
-import recordEvent from '../../monitoring/record-event';
-import localStorage from '../../utilities/storage/localStorage';
-import { fetchAndUpdateSessionExpiration as fetch } from '../../utilities/api';
+import { apiRequest } from '../../utilities/api';
 import { inProgressApi } from '../helpers';
 import { VA_FORM_IDS_SKIP_INFLECTION } from '../constants';
 
 export function removeFormApi(formId) {
-  const csrfTokenStored = localStorage.getItem('csrfToken');
   const apiUrl = inProgressApi(formId);
-  return fetch(apiUrl, {
+  return apiRequest(apiUrl, {
     method: 'DELETE',
-    credentials: 'include',
-    headers: {
-      'Content-Type': 'application/json',
-      'X-Key-Inflection': 'camel',
-      'Source-App-Name': window.appName,
-      'X-CSRF-Token': csrfTokenStored,
-    },
-  })
-    .then(res => {
-      if (!res.ok) {
-        return Promise.reject(res);
-      }
-
+  }).catch(error => {
+    if (error instanceof Error) {
+      Sentry.captureException(error);
+      Sentry.captureMessage('vets_sip_error_delete');
       return Promise.resolve();
-    })
-    .catch(res => {
-      if (res instanceof Error) {
-        Sentry.captureException(res);
-        Sentry.captureMessage('vets_sip_error_delete');
-        return Promise.resolve();
-      } else if (!res.ok) {
-        Sentry.captureMessage(`vets_sip_error_delete: ${res.statusText}`);
-      }
+    }
 
-      return Promise.reject(res);
-    });
+    Sentry.captureMessage(`vets_sip_error_delete: ${error.statusText}`);
+    return Promise.reject(error);
```

Comment:
It looks like `removeFormApi` can still return with a Promise rejection...

Reply:
Yes, it can.  `apiRequest` will include the rejected Promise which we use to look at the Response (status code, etc) as well as any network errors.